marklogic-data-hub/build.gradle

```diff
@@ -31,15 +31,15 @@ ext.junitJupiterVersion  = '5.3.1'
 
 dependencies {
```

Comment:
In Line 6, in this file, I still see ml-gradle as 3.15.1. Should that be updated as well ?

Reply:
Yes, it might as well - that ml-gradle is just used for deploying the DHF test application, but no reason not to update it. Thanks.